e2e-tests/src/test/rewards.rs

```diff
@@ -311,3 +304,82 @@ pub fn force_new_era(config: &Config) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+pub fn change_stake_and_force_new_era(config: &Config) -> anyhow::Result<()> {
+    const MAX_DIFFERENCE: f64 = 0.07;
```

Comment:
1) I can see all tests in this file re-use the same value, does it make sense to extract it to a common value in this file (e.g. making `mod rewards` with that const)?
2) this is a relative difference right? and exactly from what two values? I can see it is used in `check_rewards` to compare some fractions. Maybe we can add some comment describing what we are actually comparing with that const?